var/spack/repos/builtin/packages/blender/package.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2013-2018 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+
+from spack import *
+
+import os
+import shutil
+
+class Blender(Package):
+    """Blender is the free and open source 3D creation suite."""
+
+    homepage = "https://www.blender.org"
+
+    version('2.81a-217', sha256='bb6e03ef79d2d7273336f8cfcd5a3b3f')
+
+    conflicts("python")
+    conflicts('platform=darwin', msg='this binary release of blender requires Linux')
+    conflicts('platform=ppc64', msg='this binary release of blender requires Linux')
+
+
+    def install(self, spec, prefix):
+	for file in os.listdir(self.stage.source_path):
```

Comment:
??
==> Error: invalid syntax in /Users/leite/dev/spack/var/spack/repos/builtin/packages/blender/package.py, line 23

/Users/leite/dev/spack/var/spack/repos/builtin/packages/blender/package.py:22:48: E999 TabError: inconsistent use of tabs and spaces in indentation